core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -106,7 +106,7 @@ export class Jazz {
     return element(by.xpath('//div[contains(text(),"successful")]'));
   }
   getAssetStatusVerify() {
-    return element(by.xpath('//div[@class="det-value"]'));
+    return element(by.xpath('//div[contains(text(),"ACTIVE")]'));
```

Comment:
Wherever available are we using specific id, class, title etc. instead of contains in the xpath search string?